### PR TITLE
[Imporve] Optimize log printing in GlobalExceptionHandler

### PR DESCRIPTION
**Pull request by caicancai**
## What changes were proposed in this pull request

Issue Number: close #xxx 

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (yes / no)

### CODE REVIEW

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/handler/GlobalExceptionHandler.java [hunk 1]
```diff
@@ -113,6 +113,6 @@ public RestResponse handleConstraintViolationException(ConstraintViolationExcept
   @ExceptionHandler(value = UnauthorizedException.class)
   @ResponseStatus(HttpStatus.FORBIDDEN)
   public void handleUnauthorizedException(Exception e) {
-    log.info("Permission denied，{}", e.getMessage());
+    log.error("Permission denied，{}", e.getMessage());
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/handler/GlobalExceptionHandler.java]: If e is NullPointerException, then e.getMessage will throw nullPointExp...  You can test this:

```
  @Test
  public void testException() {
    try {
      throw new NullPointerException();
    } catch (Exception e) {
      // e.getMessage will be throw nullExp....
      System.out.printf(e.getMessage());
    }
  }
```

**Comment by wolfboys** [comment]
There are some minor issues here, You can refer to this [PR](https://github.com/apache/incubator-streampark/pull/3705)

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/handler/GlobalExceptionHandler.java (hunk 1 above)
Comment (caicancai) [reply streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/handler/GlobalExceptionHandler.java]: done

**Review by VampireAchao — APPROVED** [review]
LGTM

**Pull request merged**